specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -287,18 +287,17 @@ interoperability protocol based on RMI-IIOP.
 ==== Support for Transactional Resource Managers
 
 This specification requires all Jakarta EE products to support the
-_jakarta.transaction.xa.XAResource_ interface, as specified in the Connector
+_javax.transaction.xa.XAResource_ interface, as specified in the Connector
 specification. This specification also requires all Jakarta EE products to
-support the _jakarta.transaction.xa.XAResource_ interface for performing two-phase
+support the _javax.transaction.xa.XAResource_ interface for performing two-phase
 commit operations on JDBC drivers that support the JTA XA APIs. This
 specification does not require that JDBC drivers or Jakarta Messaging providers
-use the _jakarta.transaction.xa.XAResource_ interface, although they may use
+use the _javax.transaction.xa.XAResource_ interface, although they may use
 this interface and in all cases they must meet the transactional resource
 manager requirements described in this chapter. In particular, it must be
 possible to combine operations on one or more JDBC databases, one or more
 Jakarta Messaging sessions, one or more enterprise beans, and multiple resource
-adapters supporting the _XATransaction_ mode in a single transaction defined by
-the Jakarta Transactions specification.
+adapters supporting the _XATransaction_ mode in a single JTA transaction.
```

Comment:
I don't think we should add "JTA" here. I am thinking XATransaction might actually be defined in Connector: https://github.com/eclipse-ee4j/jca-api/blob/6339be3f1bdacf5c0551d535790bcd78e552cd38/api/src/main/java/jakarta/resource/spi/TransactionSupport.java#L58 (@smillidge or @jmstephensgit - would you be able to comment to that please?)

Reply:
This was also present in the original document

https://github.com/eclipse-ee4j/jakartaee-platform/blame/dbb1841d0a7cabead8eb6060db23b0d8c7ffedc8/specification/src/main/asciidoc/platform/TransactionManagement.adoc#L310-L311

Reply:
I think perhaps the wording "single transaction defined by the Jakarta Transactions specification" is possibly OK so maybe removing any change around this one

Reply:
Done. Reverted to previous text.